docs/_sass/_style.scss

```diff
@@ -1031,3 +1031,11 @@ code.output {
   clip: rect(0, 0, 0, 0);
   border: 0;
 }
+
+.result {
+  border: 1px solid yellow;
```

Comment:
I'd rather simply apply `highlight` for now. We can discuss style in a separate PR.

![result-styles](https://cloud.githubusercontent.com/assets/103008/21732810/9fd76e98-d45b-11e6-9e2c-8bb8f3b3a4ce.png)